pkg/omreport/util.go

```diff
@@ -75,6 +75,82 @@ func severity(s string) string {
 	return "0"
 }
 
+func pdiskState(s string) string {
+	states := map[string]string{
```

Comment:
Would you be up to document these states in the documentation?
So others could more easily build their monitoring around these new metrics.

Reply:
Ah, yeah, I knew there was something I forgot lol. I'll get on that shortly

Reply:
Thanks, really appreciate your PR!